examples/pinterest.amp.html

```diff
@@ -458,5 +459,19 @@
       data-description="Next stop: Pinterest">
     </amp-pinterest>
 
+  <h4>Follow Button</h4>
+
+    <amp-pinterest height=20 width=87
+      data-do="buttonFollow"
+      data-href="https://www.pinterest.com/zackargyle/"
```

Comment:
Also in the [amp-pinterest spec](https://github.com/ampproject/amphtml/blob/master/extensions/amp-pinterest/amp-pinterest.md), the `data-url` attribute is described, but not the `data-href`. We should make sure these are in sync.